scripts/hudson/quickstart.sh

```diff
@@ -22,8 +22,9 @@ function fatal {
 }
 
 [ $WORKSPACE ] || fatal "please set WORKSPACE to the quickstarts directory"
-NARAYANA_REPO=jbosstm
-NARAYANA_BRANCH="master"
+NARAYANA_REPO=${NARAYANA_REPO:-jbosstm}
+NARAYANA_BRANCH="${NARAYANA_BRANCH:-master}"
+NARAYANA_VERSION=${NARAYANA_VERSION:-}
```

Comment:
You could just say: NARAYANA_VERSION=${NARAYANA_VERSION:-5.7.2.Final-SNAPSHOT} as the release script does do a search and replace in *.sh anyway so that would be kept up-to-date.

Later on you would be able to set NARAYANA_VERSION_PARAM with -Dversion.narayana=${NARAYANA_VERSION} directly.

What do you think?

Reply:
Ok, thanks for the changeset. I will do so.